css/fonts.css

```diff
@@ -0,0 +1,31 @@
+@font-face {
+  font-family: 'Montserrat';
+  src: url('./fonts/montserrat-bold-webfont.woff2') format('woff2'),
+       url('./fonts/montserrat-bold-webfont.woff') format('woff');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  src: url('./fonts/montserrat-medium-webfont.woff2') format('woff2'),
+       url('./fonts/montserrat-medium-webfont.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/finalproject/pullRequest?prid=5923514)